### PR TITLE
Add Support for Redis Version 7.0.4

### PR DESCRIPTION
**Pull request by Shaad7**
Signed-off-by: Shaad7 <abdullah.alshaad@appscode.com>

### CODE REVIEW

File: catalog/raw/redis/redis-7.0.4.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  exporter:
+    image: kubedb/redis_exporter:1.9.0
+  initContainer:
+    image: kubedb/redis-init:0.5.0
+  podSecurityPolicies:
+    databasePolicyName: redis-db
+  stash:
+    addon:
+      backupTask:
+        name: redis-backup-6.2.5
```
Comment (hossainemruz) [inline catalog/raw/redis/redis-7.0.4.yaml:19]: Use `redis-backup-7.0.4`

File: catalog/raw/redis/redis-7.0.4.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      restoreTask:
+        name: redis-restore-6.2.5
```
Comment (hossainemruz) [inline catalog/raw/redis/redis-7.0.4.yaml:21]: Use `redis-restore-7.0.4`

**Review by hossainemruz — CHANGES_REQUESTED** [review]
Please update the task version and regenerate the chart.

File: catalog/raw/redis/redis-7.0.4.yaml (hunk 2 above)
Comment (Shaad7) [reply catalog/raw/redis/redis-7.0.4.yaml]: Updated

File: catalog/raw/redis/redis-7.0.4.yaml (hunk 1 above)
Comment (Shaad7) [reply catalog/raw/redis/redis-7.0.4.yaml]: Updated

**Review by hossainemruz — APPROVED** [review]

**Pull request merged**